ethcore/snapshot/src/lib.rs

```diff
@@ -439,19 +431,17 @@ impl StateRebuilder {
 
 			for (hash, thin_rlp) in pairs {
 				if !flag.load(Ordering::SeqCst) { return Err(Error::RestorationAborted.into()) }
-
-				if &thin_rlp[..] != &empty_rlp[..] {
-					self.bloom.set(hash.as_bytes());
-				}
 				account_trie.insert(hash.as_bytes(), &thin_rlp)?;
 			}
 		}
 
-		let bloom_journal = self.bloom.drain_journal();
-		let mut batch = backing.transaction();
-		StateDB::commit_bloom(&mut batch, bloom_journal)?;
-		self.db.inject(&mut batch)?;
-		backing.write_buffered(batch);
+		// todo[dvdplm] double-check that this is ok to remove – where are all the above db changes written? In caller? Unsure about what `inject` actually does.
+		// let backing = self.db.backing().clone();
+		// let bloom_journal = self.bloom.drain_journal();
+		// let mut batch = backing.transaction();
+		// StateDB::commit_bloom(&mut batch, bloom_journal)?;
+		// self.db.inject(&mut batch)?;
+		// backing.write_buffered(batch);
```

Comment:
I think it's fine to remove, the only batch we're inserting here is bloom changes.